geode-docs/tools_modules/micrometer/micrometer-configuration.html.md.erb

```diff
@@ -0,0 +1,120 @@
+---
+title:  Configuration and Publishing
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+
+# Meter configuration
+
+Out of the box <%=vars.product_name%> is instrumented and can emit meters with a properly created Meter Registry. 
+
+However, any meters that are timers based on the system clock time, similar to the time statistics, they will obey the existing `enable-time-statistics` <%=vars.product_name%>.properties setting, see **[Setting up Statistics](../../managing/statistics/setting_up_statistics.html)**.
```

Comment:
Technically this is only true for the `geode.cache.gets` timer. The `geode.function.executions` timer will currently ignore this setting and will always record time.

Reply:
We should probably submit another code PR that makes all meters consistent in this regard.

Reply:
Consider removing the period betweeen `<%=vars.product_name%>` and `properties`. It looks strange in the generated docs because the product name is capitalized.

Reply:
> We should probably submit another code PR that makes all meters consistent in this regard.

Agreed.

Reply:
> Consider removing the period betweeen `<%=vars.product_name%>` and `properties`. It looks strange in the generated docs because the product name is capitalized.

Just changed it to hardcoded lowercase 'geode'. Don't need the variable in this case.